activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Add `assert_differences` to `ActiveSupport::Testing::Assertions`
```

Comment:
Only 1 other has a period at the end.  Should I add periods to all while I'm at it?